libs/test-common/src/main/java/org/codice/ddf/test/common/annotations/PaxExamRule.java

```diff
@@ -105,6 +107,17 @@ private void starting(Description description) {
       failRule(testClassName, throwable, BEFORE_EXAM_FAILURE_MESSAGE);
     }
 
+    if (firstSuiteRun) {
+      try {
+        firstSuiteRun = false;
+        runAnnotations(BeforeSuite.class, testClass);
+      } catch (Throwable throwable) {
+        setupFailed = true;
+        LOGGER.error("Failed to setup SUITE for " + testClassName, throwable);
```

Comment:
❔ should we be using parameterized logging here? Can't remember on error messages.

Reply:
We can't in this case because we're passing in the `Throwable`.